pkgs/applications/radio/tncattach/default.nix

```
@@ -0,0 +1,28 @@
+{ lib, stdenv, fetchFromGitHub, installShellFiles }:
+
+stdenv.mkDerivation rec {
+  pname = "tncattach";
+  version = "0.1.9";
+
+  src = fetchFromGitHub {
+    owner = "markqvist";
+    repo = "tncattach";
+    rev = "bb9ff10158c04a37cb3221152885a083e3059909";
+    sha256 = "0n7ad4gqvpgabw2i67s51lfz386wmv0cvnhxq9ygxpsqmx9aynxk";
+  };
+
+  nativeBuildInputs = [ installShellFiles ];
+
+  installPhase = ''
+    install -Dm755 tncattach -t $out/bin
+    installManPage tncattach.8
+  '';
```

One more minor thing:
```suggestion
  installPhase = ''
    runHook preInstall
    install -Dm755 tncattach -t $out/bin
    installManPage tncattach.8
    runHook postInstall
  '';
```